### DISCUSSION TITLE
Extra blank page after updating birt

### DISCUSSION THREAD

**Discussion by tameergh1** [eclipse-birt/birt#1744] [General]
After updating BIRT from 4.8 to 4.15 an extra blank page is added after each chart.
Could it be something else other thank "pageBreakAfter" & "pageBreakBefore" props?

**Comment by speckyspooky** [comment]
Which output we are talking, HTML, PDF, WORD?
I created a small report with a chart on my side but I got no page break after a chart not on the preview (HTML) nor at PDF.

Could you provide a simple example report with your issue?
Then we can investigate there better and have the same starting point.

(If you create the test example please do it without external sources so that it is runable for debugging.)

**Reply by tameergh1** [reply]
It is complicated to provide simple example, but what can cause extra blank page other than the two props mentioned above?

**Reply by hvbtup** [reply]
For example, a footer line, even if it's empty.
